tensorbay/opendataset/HKD/loader.py

```diff
@@ -5,7 +5,6 @@
 # pylint: disable=invalid-name
 
 """Dataloader for Finger Tapping Hand Keypoints Dataset(HKD)."""
-
```

Comment:
why remove this line?

Reply:
it is unintentional. it might occurs when I modify the code at a random time.

Reply:
just revert it back

Reply:
ok